scripts/cloudlab/start_onenode_vhive_cluster.sh

```diff
@@ -26,6 +26,17 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 ROOT="$( cd $DIR && cd .. && cd .. && pwd)"
 SCRIPTS=$ROOT/scripts
 
+SANDBOX=$1
+
+if [ -z "$SANDBOX" ]; then
+    SANDBOX="firecracker"
+fi
+
+if [ "$SANDBOX" != "gvisor" ] && [ "$SANDBOX" != "firecracker" ]; then
+    echo Specified sanboxing technique is not supported. Possible are \"firecracker\" and \"gvisor\"
+    exit 1
+fi
+
 echo Clean up host resources if left after previous runs
 $SCRIPTS/github_runner/clean_cri_runner.sh
```

Comment:
Pass gvisor as an argument to clean_cri_runner and add a case for the same [here](https://github.com/nathanieltornow/vhive/blob/d04376524de4c83afd1a54e6149f492efcc97a71/scripts/github_runner/clean_cri_runner.sh#L30).